src/ui/layout/search/SearchResultsDropDown.vue

```diff
@@ -91,12 +104,18 @@ export default {
         previewChanged(changedPreviewState) {
             this.previewVisible = changedPreviewState;
         },
-        showResults(passedAnnotationResults, passedObjectResults) {
-            if ((passedAnnotationResults && passedAnnotationResults.length)
-                || (passedObjectResults && passedObjectResults.length)) {
+        showSearchStarted() {
+            this.searchLoading = true;
+            this.resultsShown = true;
+            this.annotationResults = [];
+            this.objectResults = [];
+        },
+        showResults(searchLoading, passedSearchQuery, passedAnnotationResults, passedObjectResults) {
+            this.searchLoading = searchLoading;
+            this.annotationResults = passedAnnotationResults;
+            this.objectResults = passedObjectResults;
+            if (passedSearchQuery && passedSearchQuery.length) {
```

Comment:
We can also use optional chaining here:

```suggestion
            if (passedSearchQuery?.length) {
```